cve_bin_tool/extractor.py

```diff
@@ -178,17 +178,25 @@ async def extract_file_cab(filename, extraction_path):
         return 0
 
     @staticmethod
-    async def extract_file_zip(filename, extraction_path, process_can_fail=False):
+    async def extract_file_zip(filename, extraction_path, process_can_fail=True):
         """Extract zip files"""
+
+        is_exe = filename.endswith(".exe")
         if await aio_inpath("unzip"):
             stdout, stderr, _ = await aio_run_command(
-                ["unzip", "-n", "-d", extraction_path, filename]
+                ["unzip", "-n", "-d", extraction_path, filename], process_can_fail
             )
             if stderr or not stdout:
+                if is_exe:
```

Comment:
return (int(not is_exe))
Take it or leave it, just saves a couple of lines, but not great readability.